src/core/watchQueryOptions.ts

```diff
@@ -176,6 +178,13 @@ export interface WatchQueryOptions<
 
   /** {@inheritDoc @apollo/client!QueryOptions#canonizeResults:member} */
   canonizeResults?: boolean;
+
+  /**
+   * A callback function that's called whenever a refetch attempt occurs
+   * while polling. If the function returns `false`, the refetch is
+   * skipped and not reattempted until the next poll interval.
+   */
+  shouldRefetchOnPolling?: ShouldRefetchOnPolling;
```

Comment:
```suggestion
  /**
   * A callback function that's called whenever a refetch attempt occurs
   * while polling. If the function returns `true`, the refetch is
   * skipped and not reattempted until the next poll interval.
   */
  skipPollAttempt?: SkipPollAttemptFn;
```

Ok, I've got a final answer that we've all agreed on for the name of this function. We'd like to call this `skipPollAttempt`, and the behavior will be inverse what we have here. If this function returns `true`, the poll attempt will be skipped, otherwise the network request will be executed. Would you mind making these updates to name it as such?

Reply:
```suggestion
  /**
   * A callback function that's called whenever a refetch attempt occurs
   * while polling. If the function returns `false`, the refetch is
   * skipped and not reattempted until the next poll interval.
   */
  shouldRefetchOnPolling?: () => boolean;
```

How do you feel about inlining the function type here? We do this for some other types (such as [`onError`](https://github.com/apollographql/apollo-client/blob/aaf990bd2816a155801926517b4d60f26f0b2a95/src/core/watchQueryOptions.ts#L219)), so this would still follow convention from some of the other types.